### ISSUE TITLE
CapitalizationInSelector linter incorrectly reports lint for capitalized attribute selectors

### ISSUE THREAD

**Issue by joaomilho** [brigade/scss-lint#40]
I'm having a "Selectors should be lowercase" in the following line:

``` css
&.small[rel=replaceCard] {
```
